README.md

```diff
@@ -17,7 +17,7 @@ If you have an existing React Native project, it's easy to begin using FluentUI
 
 ### Prerequisites
 
-- [Standard React Native dependencies](https://reactnative.dev/docs/environment-setup)
+- [Standard React Native dependencies](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies#manual-setup)
```

Comment:
Does this link also link the reactnative environment setup?

Reply:
No, I just added another item in the list with the environment setup, though.